### PR TITLE
guile-json: init at 4.4.1

### PR DESCRIPTION
**Pull request by ethancedwards8**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Review by MagicRB — APPROVED** [review]
Looks fine too

**Comment by ethancedwards8** [comment]
It builds fine on my machine. Not sure why ofborg fails

File: pkgs/development/guile-modules/guile-json/default.nix [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  meta = with lib; {
+    description = "JSON Bindings for GNU Guile";
+    homepage = "https://savannah.nongnu.org/projects/guile-json";
+    license = licenses.gpl3Plus;
+    maintainers = with maintainers; [ ethancedwards8 ];
+    platforms = platforms.linux;
+  };
+}
+
+
```
Comment (SuperSandro2000) [inline pkgs/development/guile-modules/guile-json/default.nix:41]:
```suggestion
}
```

**Comment by SuperSandro2000** [comment]
> It builds fine on my machine. Not sure why ofborg fails

ofborg does not fail.

**Comment by ethancedwards8** [comment]
Fixed. Thanks.

**Pull request merged**